source/hassio/debugging.markdown

```diff
@@ -34,7 +34,7 @@ Make sure when you are copying the public key to the root of the SD card that yo
 journalctl -f -u resin-supervisor.service
 
 # Hass.io supervisor logs
-docker logs hassio_supervisor
+docker logs resin_supervisor
```

Comment:
I think there might be an actual problem with the setup. I downloaded hass.io and I immediately had an update for the supervisor. I only tried to look at logs after the update and the command supplied in the docs didn't match.